File: app/src/main/java/com/example/getfood/ui/loginregister/LoginActivityMvpView.java
package com.example.getfood.ui.loginregister;

import android.content.Context;

import com.example.getfood.ui.base.BaseView;

public interface LoginActivityMvpView extends BaseView {

    Context getContext();

    void onSuccessfulVerificationAndSignIn();

    void onFailedVerificationOrSignIn(Exception e);

    void onTokenUpdatedSuccessfully();
}

File: app/src/main/java/com/example/getfood/ui/loginregister/LoginActivityMvpView.kt
package com.example.getfood.ui.loginregister

import android.content.Context
import com.example.getfood.ui.base.BaseView

open interface LoginActivityMvpView : BaseView {
    val context: Context
    fun onSuccessfulVerificationAndSignIn()
    fun onFailedVerificationOrSignIn(e: Exception)
    fun onTokenUpdatedSuccessfully()
}